source/views/directory/detail-android.js

```diff
@@ -93,11 +95,11 @@ export function DirectoryDetailView(props: Props) {
 					))}
 				</View>
 
-				{officeHours ? (
+				{officeHours && (officeHours.prefix || officeHours.content) ? (
```

Comment:
You should be able to just check if `officeHours` is null; the API will return `null` if the person doesn't have anything set.

Reply:
It’s more that I’m finding myself wanting to know if these individual props have content or not.